analyzer/codechecker_analyzer/analyzer_context.py

```diff
@@ -11,7 +11,7 @@
 
 
 # pylint: disable=no-name-in-module
-from distutils.spawn import find_executable
+from shutil import which
```

Comment:
These should be sorted alphabetically, and check whether the pylint silencing of the warning still applies.